### PR TITLE
Reformat autodocs with word-wrapping head comments

### PR DESCRIPTION
**Pull request by tempoz**
This still isn't perfect, since the example comments sometimes exceed the bounds of the text box, but it's better.

---

**Version bump**: Patch

### CODE REVIEW

**Comment by tempoz** [comment]
Current dependencies on/for this PR:
* master
  * **PR #2215** <a href="https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/2215" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/2215?utm_source=stack-comment).

**Review by bduffany — APPROVED** [review]
nice! just nits / sanity checks

File: server/util/flagutil/yaml/yaml.go [hunk 1]
```diff
@@ -32,6 +33,28 @@ var (
 	}
 
 	AppendTypeToLineComment = &appendTypeToLineComment{}
+
+	WordWrapForNDots = []*regexp.Regexp{
+		regexp.MustCompile("(([^\n]{1,76})((\\s+)|$))"),
```
[the remark this answers is not in the document]
Comment (tempoz) [reply server/util/flagutil/yaml/yaml.go:38]: It is 76; I counted the width of the website window. You understand the regex correctly :P

File: server/util/flagutil/yaml/yaml.go [hunk 2]
```diff
@@ -32,6 +33,28 @@ var (
...
+		regexp.MustCompile("(([^\n]{1,40})((\\s+)|$))"),
+		regexp.MustCompile("(([^\n]{1,36})((\\s+)|$))"),
+		regexp.MustCompile("(([^\n]{1,32})((\\s+)|$))"),
+		regexp.MustCompile("(([^\n]{1,28})((\\s+)|$))"),
+		regexp.MustCompile("(([^\n]{1,24})((\\s+)|$))"),
+		regexp.MustCompile("(([^\n]{1,20})((\\s+)|$))"),
+		regexp.MustCompile("(([^\n]{1,16})((\\s+)|$))"),
+		regexp.MustCompile("(([^\n]{1,12})((\\s+)|$))"),
+		regexp.MustCompile("(([^\n]{1,8})((\\s+)|$))"),
+		regexp.MustCompile("(([^\n]{1,4})((\\s+)|$))"),
```
[the remark this answers is not in the document]
Comment (tempoz) [reply server/util/flagutil/yaml/yaml.go:56]: I almost did that and then I thought, "Regexes are hard enough to read when you're looking at the literal string, generated regexes would be even more confusing." I am on the fence about it, though; obviously, I don't love all the duplicated code, and it seems like you tripped over the number of dots thing, so probably better to generate it with a function.

[the remark this answers is not in the document]
Comment (tempoz) [reply server/util/flagutil/yaml/yaml.go]: Done!

[the remark this answers is not in the document]
Comment (bduffany) [reply server/util/flagutil/yaml/yaml.go]: The generated code is very hard to understand, but on the other hand the regexes are so opaque that I was already treating them as a black box. If we want to modify the wrapping behavior other than by changing the target length, we can always just hand-write a wrapping func.

**Review by bduffany — APPROVED** [review]

**Pull request merged**